### PR TITLE
finish half completed interner documentation

### PR DESCRIPTION
**Pull request by loftyinclination**

### CODE REVIEW

**Review by Byron — DISMISSED** [review]
Thanks a lot, much appreciated.

Please note that I further tuned the description.

Edit: and I realized that I can't merge if I am the last one to have pushed, fair enough.

File: src/intern.rs [hunk 1]
```diff
@@ -104,13 +103,13 @@ pub struct Interner<T: Hash + Eq> {
 }
 
 impl<T: Hash + Eq> Interner<T> {
-    /// Create an Interner with an intial capacity calculated by calling
+    /// Create an Interner with an initial capacity calculated by calling.
```
Comment (loftyinclination) [inline src/intern.rs:106]: not sure that there should be a period here

Comment (Byron) [reply src/intern.rs:106]: Right, it's not actually complete. Please feel free to finish it ,maybe then I can merge as well. Pushing into this branch should suffice.

Comment (Byron) [reply src/intern.rs:106]: @loftyinclination Could you make the changes? It really needs another commit that isn't mine so I can merge. Thank you.

Comment (pascalkuthe) [reply src/intern.rs:106]: feel free to push yourself @Byron  and I can merge

Comment (loftyinclination) [reply src/intern.rs]: sorry for the delay, just pushed

**Review by pascalkuthe — APPROVED** [review]

**Pull request merged**